src/handlers/StatesBtn.ts

```
@@ -0,0 +1,113 @@
+}
+
+// managers should be able to load and save their states as well
+@Gated(CheckRights())
+@ButtonRoute(SaveState)
+export class SaveStateBtn extends ButtonHandler<[typeof SaveState]> {
+    public async execute(): Promise<void> {
+        const channel = this.event.channel
+        if (!channel?.isVoiceBased()) return
+        const settings = await database.findChannel(channel.id)
+        if (!settings ||
+            settings.closed == null ||
```

ESLint

[error] Use '===' to compare with null.

(no-eq-null)